### DISCUSSION TITLE
Compact table

### DISCUSSION THREAD

**Discussion by ryanchesler** [h2oai/wave#1774] [Q&A] [answered]
Is there a way to make the columns slightly more compact or change the default width? I have a table with several columns I want to show. Many of them with booleans or single digit floats and they take up so much space because of the default width of table columns. Causes a lot of horizontal scrolling. If I could make these columns narrower then I could fit much more on the screen. Is there any option to change this somehow?

**Comment by ryanchesler** [comment, marked as answer]
Found the option. min_width for ui.table_column https://wave.h2o.ai/docs/api/ui#table_column-

**Reply by mturoci** [reply]
Yep, that's the one you can use to control the column width. There is also max_width btw.
